amgut/db/patches/0029.sql

```diff
@@ -0,0 +1,155 @@
+-- October 12, 2015
+-- Add tables needed to track plate maps and create prep templates
+--Need to wipe out old, bad data in the plate table
+DROP TABLE barcodes.plate_barcode CASCADE;
+DELETE FROM barcodes.plate;
+
+ALTER TABLE barcodes.barcode ALTER COLUMN barcode TYPE varchar;
+ALTER TABLE barcodes.project ADD description varchar  NOT NULL DEFAULT '';
+
+CREATE TABLE barcodes.lot ( 
+	lot_id               bigserial  NOT NULL,
+	lot_type             varchar  NOT NULL,
+	lot                  varchar  NOT NULL,
+	date_added           date DEFAULT current_date NOT NULL,
+	CONSTRAINT pk_lots PRIMARY KEY ( lot_id )
+ );
+
+CREATE TABLE barcodes.robots ( 
+	robot_id             bigserial  NOT NULL,
+	robot_type           varchar  NOT NULL,
+	robot_name           varchar(100)  NOT NULL,
```

Comment:
this is effectively the ID; is there a reason to not use it as it's a natural pk?